ncolony/tests/test_ctllib.py

```diff
@@ -13,6 +13,11 @@
 
 from ncolony import ctllib
 
+def jsonFrom(fname):
+    """Load JSON from a file"""
+    with open(fname, "rb") as fp:
```

Comment:
Why not `io.open(fname, 'r', encoding='utf-8')`?  Then you can do `json.load(fp)`.

Reply:
done